src/content/docs/en/reference/api-reference.mdx

```diff
@@ -797,7 +914,7 @@ const enterprise = await getEntryBySlug('blog', 'enterprise');
 
 ### Collection Entry Type
 
-The [`getCollection()`](#getcollection) and [`getEntryBySlug()`](#getentrybyslug) functions each return entries with the `CollectionEntry` type. This type is available as a utility from `astro:content`:
+Query functions including [`getCollection()`](#getcollection) and [`getEntry()`](#getentry) each return entries with the `CollectionEntry` type. This type is available as a utility from `astro:content`:
```

Comment:
I know you've said "including.." so this doesn't need to be a comprehensive list, but I'll also note that `getEntries()` is not mentioned here.